### ISSUE TITLE
use esbuild instead of @preset/react to transform to jsx

### ISSUE THREAD

**Issue by Leonewu** [hmsk/vite-plugin-markdown#187]
I just read vite's code about transfroming jsx syntax.It uses esbuild.
And when I install vite-plugin-markdown then run dev server, it thorwed an error 'module @babel/preset-react not found'.
I dive into the code in this plugin and find it uses babel to transform jsx syntax.
I think it is not neccessary to use babel because vite uses esbuild internally.

**Comment by hmsk** [comment]
good point. this plugin was originally started by copying frontmatter-markdown-loader which is for Webpack, so React's build was still relying on Babel. Decoupling sounds natural choice to me and I'm going to work on someday but use `@babel/core` and `@babel/preset-react` atm if you are eager to use this plugin.

**Comment by lebbe** [comment]
Would you accept PRs resolving this issue?

**Comment by hmsk** [comment]
@lebbe yes if it's compatible with current one! thanks for your offering.
